src/pages/hegel/reference/the-mechanical-process/the-formal-mechanical-process-section-1.mdx

```diff
@@ -0,0 +1,243 @@
+---
+title: The Formal Mechanical Process, Section 1
+description:
+    Learn about the first section of the Formal Mechanical Process from Hegel's
+    Science of Logic
+isArticle: true
+authors: Ahilleas Rokni (2024)
+---
+
+## (a) The Formal Mechanical Process: Section 1
+
+The first section of B. The Mechanical Process that deals with the logical
+development of Mechanism is (a) The Formal Mechanical Process.
+
+The section begins with the following: “The mechanical process is the positing
+of what is contained in the Notion of mechanism, and therefore, in the first
+instance, of a _contradiction_.” (Hegel 1991, 716). The “concept” of Mechanism
+is the logical moment with which the section, A. The Mechanical Object finished.
+The `mechanical object` is now a negative unity: the negativity of having the
+identical determinateness to other mechanical objects whilst being indifferent
+to them. This is the contradiction with which A. The Mechanical Object
+concludes. This contradiction is now explored further in the (a) The Formal
+Mechanical Process.
+
+Above, Hegel writes that it is the “positing” of the contradiction in the
+`mechanical object` that gets the mechanical process started and it is worth
+dwelling on this for a moment. Hegel explains that by “positing” he means,
+
+> It follows from the Notion just indicated that the interaction of objects
+> takes the form of the _positing_ of the _identical_ relation of the objects.
+> This consists merely in giving to the determinateness that is determined, the
+> form of _universality_; this is _communication_, which does not involve
+> transition into an opposite (Hegel 1991, 716) Thus, the “positing” is an
+> expression of the logical interaction of objects that relate to each other as
+> identical. But where has this logical interaction come from? If we cast our
+> minds back to the conclusion of A. The Mechanical Object we will recall that
```

Comment:
Here you can insert a link. Also, given articles are essentially self-contained, you can't assume that readers will have followed the entire development up till now (most likely they will, but still), so I'd rephrase the "cast our minds back to..." bit.